eeps/eep-0048.md

```diff
@@ -152,9 +154,10 @@ For each entry in `Docs`, we have:
     may be rendered as as a single line or two lines. It exists
     exclusively for exhibition purposes
 
-  * `Doc` - a binary with the documentation or the atom `none`
-    in case there is no documentation or the atom `hidden` if
-    documentation has been explicitly disabled for this entry
+  * `Doc` - a map with the documentation language as key, such as
+    "en" or "pt_BR", and the documentation as a binary value.  It may be
```

Comment:
Is the key a string or a binary.  If binary and if this is Erlang Term format the examples should be `<<"en">>` and `<<"pt_BR">>`, right?